### PR TITLE
Bug 2002588: Use sentence case

### PR DESCRIPTION
**Pull request by yaacov**
Use sentence case

Before:
screenshot-localhost_9000-2021 09 13-11_46_59

After:
screenshot-localhost_9000-2021 09 13-11_45_50

### CODE REVIEW

**Comment by yaacov** [comment]
@avivtur @glekner @metalice @pcbailey please review

File: frontend/packages/kubevirt-plugin/locales/en/kubevirt-plugin.json [hunk 1]
```diff
@@ -635,8 +635,8 @@
   "Total Bytes": "Total Bytes",
   "Used Bytes": "Used Bytes",
   "FileSystems": "FileSystems",
-  "No File Systems Found": "No File Systems Found",
-  "File Systems": "File Systems",
+  "No File systems found": "No File systems found",
```
Comment (pcbailey) [inline frontend/packages/kubevirt-plugin/locales/en/kubevirt-plugin.json:638]: "File" should be lowercase as well.

**Review by pcbailey — COMMENTED** [review]
I just looked and we really need better consistency on letter case throughout console.

File: frontend/packages/kubevirt-plugin/locales/en/kubevirt-plugin.json (hunk 1 above)
Comment (yaacov) [reply frontend/packages/kubevirt-plugin/locales/en/kubevirt-plugin.json]: Thanks, missed  that, fixed

**Pull request merged**